wrapanapi/systems/ec2.py

```diff
@@ -1187,3 +1246,45 @@ def get_registry_data(self):
                     'registry': first_registry['proxyEndpoint']}
         except (IndexError, KeyError):
             raise NotFoundError("couldn't get registry details. please check environment setup")
+
+    def create_network(self, cidr_block='10.0.0.0/16'):
+        try:
+            response = self.ec2_connection.create_vpc(CidrBlock=cidr_block)
+            network_id = response.get('Vpc').get('VpcId')
```

Comment:
Possible these keys won't be set? `response.get('Vpc', {}).get('VpcId')` would avoid AttributeError.

Reply:
Actually these keys are required. So it either will fail on create_vpc or work correctly.